### ISSUE TITLE
I cannot seem to connect to grumble

### ISSUE THREAD

**Issue by hungrymonkey** [mumble-voip/grumble#13]
Although it maybe PEBCAK, I am not really sure why I cannot connect to the server with the official client. I do not really use mumble.

I am using this script to create a cloud instance with debian 9.

The script should output an external ip address

```#!/usr/bin/env bash
INSTANCE_NAME="my-mumber-server3"
IMAGE_FAMILY="debian-9"
IMAGE_PROJECT="debian-cloud"
VM_ZONE="us-west1-a"

##zones generated $gcloud compute zones list
##https://cloud.google.com/compute/docs/regions-zones/regions-zones

gcloud compute instances create $INSTANCE_NAME \
    --image-family $IMAGE_FAMILY \
    --image-project $IMAGE_PROJECT \
    --zone $VM_ZONE \
    --metadata-from-file startup-script=./deb_startup.sh

##connecting to instances https://cloud.google.com/compute/docs/instances/connecting-to-instance
##gcloud compute ssh [instance-name]```



```#!/usr/bin/env bash
##https://cloud.google.com/compute/docs/startupscript
PACKAGES="git golang"

sudo apt-get -y install $PACKAGES

export GOPATH=$HOME/go
export PATH=$PATH:$GOPATH/bin
mkdir -p $GOPATH
go get github.com/hungrymonkey/grumble/cmd/grumble
mkdir -p $HOME/.grumble
$GOPATH/bin/grumble
```
This script get called with the above script. grumble seems to get run as root.

I tried using the output ip address to connect to mumble but it does not work.

Debian9 uses golang 1.7.

I am not sure really how to configure the server so it works

**Comment by hungrymonkey** [comment]
is this suppose to happen?

```sudo lsof -i | grep 64738
grumble   8159 rice    4u  IPv6  23001      0t0  UDP *:64738 
grumble   8159 rice    6u  IPv6  23002      0t0  TCP *:64738 (LISTEN)
```

I do not see any ipv4 listen. I only have a public ipv4 addr

**Comment by hungrymonkey** [comment]
Well, it is PEBCAK. I configured debian's firewall, but I neglected google default firewall.

'''
gcloud compute firewall-rules create allow-mumble \
     --direction=INGRESS --network=default \
     --allow=tcp:64738,udp:64738 \
     --source-ranges=0.0.0.0/0 --target-tags=mumble-tag

gcloud compute instances create my_inst \
    --tags=mumble-tag
'''

I shorten the instance create command since that is not that important as the firewall.

Basically, you have to create a firewall-rule and associated tag unless you want it to be globally applied. When you attach tags to the vm instances, the firewall-rules will also get applied too.

welcome to my daily PEBCAK. Sorry for polluting the issue on github.
